terraform/infrastructure/modules/firehose/output.tf

```diff
@@ -8,3 +8,17 @@ output "delivery_stream" {
     }
   }
 }
+
+output "firehose_subscription" {
+  value = {
+    destination = {
+      arn = aws_kinesis_firehose_delivery_stream.firehose.arn
+    }
+    role = {
+      arn = aws_iam_role.firehose_subscription.arn
+    }
+    filter = {
+      pattern = "[lambda_log_type != \"INIT_START\" && lambda_log_type != \"START\" && lambda_log_type != \"END\" && lambda_log_type != \"REPORT\", everything_else]"
```

Comment:
change `everything_else` to `everything_else_on_this_log_line`

Reply:
add a comment:

* This is a filter to remove log lines where the first item on a log line (items are space delimited) are one of INIT_START, START, etc (which are junk lambda logs)